data/campaigns/The_South_Guard/scenarios/06a_Tidings_Good_and_Ill.cfg

```diff
@@ -365,7 +365,7 @@
             message= _ "Ho! Me see elves. Elves tasty? Me take elves to eat."
         [/message]
         [message]
-            race=elf
+            speaker=second_unit
```

Comment:
Is that done anywhere else? I made the change only because it's possible to not have any elf, and the line makes the next make a (tiny) bit more sense and really could be spoken by anyone and still work.

Maybe the female and plural issue is better done as a separate sweep through the campaigns? I'm pretty sure it'll happen a lot ...